blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,281 @@
+running together, interacting with and affecting each other. It's not just about what happens
+*when*, but also *where*. There's a *spatial dimension* to account for, and it's a dimension our
+current languages are just don't designed to express.
+
+But preflights and inflights are just the tip of the iceberg in terms of what a language for the
+cloud can offer. Think about things like API endpoints and clients, telemetry, metrics and alarms,
+data schemas, and more. We have plans to bake many of these capabilities into the language and the
+standard library so that developers can truly focus on building their applications.
+
+### A standard library for the cloud
+
+The second piece of the puzzle is what we call the **Wing Cloud Library** (WCL). This is a library
+of classes that represent common resources for building cloud applications. You can think of the WCL
+as a [standard library](https://en.wikipedia.org/wiki/Standard_library) for the cloud. Similarly to
+how standard libraries in traditional languages abstract the operating system, WCL abstracts cloud
+providers and services.
+
+We believe the cloud has finally matured to the point where it is possible to establish these
+standard abstractions and that they will carry over successfully across public and private cloud
+providers. We didn't always have that level of abstraction with standard libraries. When I started
+programming, I needed to know exactly which operating system (and even which file system) I was
+using in order to write a file. Now, I can just call `writeFile()` and it will work across all
+platforms. 
+
+The WCL is designed from the ground up to support multiple targets for each resource. Classes in the
+library only define an abstract API, and it is possible to inject concrete implementations at
+compile time (technically this happens when the Wing preflight code is executed by the compiler).
+This powerful mechanism means that code written against these APIs is **portable across cloud
+providers and provisioning engines**.
+
+And of course, we think of of WCL as the foundation for an ecosystem of cloud abstractions. The
+programming model is open and extensible and anyone will be able to publish libraries and implement
+backends for existing resources based on the needs of their environment.
+
+The WCL is a major project and it's early days. You can see the coverage of resources and targets
+through this [compatibility
+matrix](https://www.winglang.io/docs/standard-library/compatibility-matrix) and make sure to "+1"
+the relevant issue if coverage is missing.
+
+Currently, WCL can only be used from Winglang code, but we are considering releasing it to other
+languages as well. The WCL is built using CDK technologies such as
+[JSII](https://github.com/aws/jsii) and [constructs](https://github.com/aws/constructs), so
+technically it is possible to publish it to other languages.
+
+### Reducing iteration time from minutes to milliseconds
```

Comment:
I think it is unexpected to talk about it here since the above two subheadings are explainng the first two main components of the wing cloud as defined above (the language and the SDK). Readers expect you to explain the third one now (the console).

Reply:
I am going to leave it like this for now. I removed the numbering from the list above so it's less triggering that we don't have affinity between the list of components and the paragraph headings.